master/reference/advanced/etcd-rbac/calico-etcdv3-paths.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Calico key and path prefixes in etcd v3
+---
+
+The Paths listed here are the key or path prefixes that a particular {{site.prodname}}
+component needs access to in etcd to function successfully.
+
+> **Note**: The path prefixes listed here may change in the future and at that point anything
+> referencing them (like etcd roles) would need to be updated appropriately.
+{: .alert .alert-info}
+
+
+## calico/node
+
+| Path                                      | Access |
+|-------------------------------------------|--------|
+| /calico/felix/v1/\*                       |   RW   |
+| /calico/ipam/v2/\*                        |   RW   |
+| /calico/resources/v3/projectcalico.org/\* |   RW   |
+
+## felix as a stand alone process
+
+| Path                                      | Access |
+|-------------------------------------------|--------|
+| /calico/felix/v1/\*                       |   RW   |
```

Comment:
What still goes in the `calico/felix/v1` path??

Reply:
That's the stuff felix writes - so endpoint status data.  I wanted to chat about that actually because I'm wondering if we should try and bring it in line - but then since the format of the data is not v2 and is only required (AFAIK) for OpenStack I'm not sure it's that important at this point.